### PR TITLE
Add native image build configuration

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject 

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3356

Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/3315
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

File: native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "methods":[
+    {"name":"getProperty","parameterTypes":["java.lang.String"] }, 
+    {"name":"setProperty","parameterTypes":["java.lang.String","java.lang.String"] }
+  ]
+},
+{
+  "name":"wso2.xmldata_graal.0.$_init",
+  "methods":[{"name":"main","parameterTypes":["java.lang.String[]"] }]
+}
+]
```
Comment (shafreenAnfar) [inline native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json:20]: I think we can format this a bit.

File: native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+[
+{
+  "name":"java.lang.String",
+  "methods":[
+    {"name":"lastIndexOf","parameterTypes":["int"] }, 
+    {"name":"substring","parameterTypes":["int"] }
+  ]
```
Comment (shafreenAnfar) [inline native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json:7]: @TharmiganK do we need the above?

Comment (TharmiganK) [reply native/src/main/resources/META-INF/native-image/io.ballerina.xmldata-native/jni-config.json:7]: I think only the resource-config is needed. Will review it before merging 👍

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/xmldata-native/resource-config.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "bundles":[{
+      "name":"MessagesBundle",
+      "locales":[""]
+    }]
+}
```
Comment (TharmiganK) [inline native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/xmldata-native/resource-config.json:6]: Can we format the json like this :
```json
{
    "bundles": [
        {
            "name": "MessagesBundle",
            "locales": [
                ""
            ]
        }
    ]
}
```

Comment (kalaiyarasiganeshalingam) [reply native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/xmldata-native/resource-config.json:10]: Formatted in [1f4e8bf](https://github.com/ballerina-platform/module-ballerina-xmldata/pull/460/commits/1f4e8bf529ea75e856301b7e6a113e5f119f083c)

**Comment by TharmiganK** [comment]
Closing this PR since this configurations should be added to ballerina lang

**Pull request closed without merging**